src/index.html

```diff
@@ -15,8 +15,35 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li><a class="nav__link blue-link is-active" href="index.html">Apple</a></li>
```

Comment:
Fix code styles everywhere 
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="index.html">Apple</a>
          </li>
```